modules/core/src/main/java/org/apache/ignite/internal/hlc/HybridTimestamp.java

```diff
@@ -230,7 +233,9 @@ public int compareTo(HybridTimestamp other) {
 
     @Override
     public String toString() {
-        return S.toString(HybridTimestamp.class, this, "physical", getPhysical(), "logical", getLogical());
+        String formattedTime = DATE_FORMATTER.format(Instant.ofEpochMilli(getPhysical()).atZone(ZoneId.systemDefault()));
+
+        return String.format("HybridTimestamp [physical=%s, logical=%d, composite=%d]", formattedTime, getLogical(), time);
```

Comment:
In general, the proposed approach looks good to me. Perhaps, `composite` is not so descriptive as `timestamp`, but again, it looks good to me.